dev-utils/karma.js

```diff
@@ -117,8 +117,8 @@ function prepareConfig(config, packageName) {
       ' (' +
       process.env.TRAVIS_BUILD_ID +
       ')'
-    config.plugins.push('karma-firefox-launcher')
-    config.browsers.push('Firefox')
+    config.plugins.push('karma-chrome-launcher')
+    config.browsers = ['ChromeHeadlessNoSandbox']
```

Comment:
this should be chrome. now that the PR is merged, our Travis tests are breaking

Reply:
I guess https://github.com/elastic/apm-agent-rum-js/pull/669 resolves it